paymentintent.go

```diff
@@ -649,6 +649,13 @@ type PaymentIntentPaymentMethodOptions struct {
 	Sofort           *PaymentIntentPaymentMethodOptionsSofort           `json:"sofort"`
 	WechatPay        *PaymentIntentPaymentMethodOptionsWechatPay        `json:"wechat_pay"`
 }
+type PaymentIntentProcessingCard struct{}
+
+// If present, this property tells you about the processing state of the payment.
+type PaymentIntentProcessing struct {
+	Card *PaymentIntentProcessingCard `json:"card"`
+	Type string                       `json:"type"`
```

Comment:
Shouldn't this be `PaymentMethodType` or an enum in general?

Reply:
Thanks Remi - it's a different set of values than PaymentMethodType (stricter limit), but agree that this should be an enum. I'll merge this and then we're going to update it to be an enum before releasing.